client/blocks/inline-help/inline-help-compact-results.jsx

```diff
@@ -1,30 +1,29 @@
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-import InlineHelpCompactResult from 'calypso/blocks/inline-help/inline-help-compact-result';
+import { decodeEntities, preventWidows } from 'calypso/lib/formatting';
+import './inline-help-search-results.scss';
 
-class InlineHelpCompactResults extends Component {
-	static propTypes = {
-		helpLinks: PropTypes.array.isRequired,
-		onClick: PropTypes.func,
-	};
-
-	static defaultProps = {
-		helpLinks: [],
-	};
-
-	render() {
-		return (
-			<ul className="inline-help__results-list">
-				{ this.props.helpLinks.map( ( link ) => (
-					<InlineHelpCompactResult
-						key={ link.link + '#' + link.id }
-						helpLink={ link }
-						onClick={ this.props.onClick }
-					/>
-				) ) }
-			</ul>
-		);
-	}
+function InlineHelpCompactResults( { helpLinks, onClick } ) {
+	return (
+		<ul className="inline-help__results-list">
+			{ helpLinks.map( ( link ) => (
+				<li key={ link.link + '#' + link.id } className="inline-help__results-item">
```

Comment:
It's not ideal that those live in 2 places. Ideally, they should be managed by a single component, and whether we'll have a "inline-help__results-cell" wrapper (and the other differences) would be determined by props.